docs/draft/6-payloads.md

```diff
@@ -0,0 +1,359 @@
+---
+permalink: /spec/6
+parent: Draft specs
+title: 6/PAYLOADS
```

Comment:
Is this meant to replace the existing 6/PAYLOADS or augment it?

Reply:
I'm not sure, but it might be worth making this a more minimal "payload-image" draft spec. Then 6/payloads could be augmented with a MAY support <image payload spec>.

Right now it is quite hard to tell what has changed, and the PR isn't mergable as is since we already have 6 marked as stable.

Reply:
Thoughts on this @cammellos?

Reply:
I don't have any opinion on this, operationally it's simpler to copy specs to draft, move, and copy over when it's stable. Visibility can be ensured by always copy a new DRAFT once a new STABLE is added (that can be a simple hook), but that also comes with some issue, for example if you partially implement a draft.
Not sure is worth though polishing the process so much as generally speaking is fairly static and we make few changes. So anything is good with me really.